Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```
@@ -114,4 +117,15 @@ select.remotion-themed::-webkit-scrollbar-thumb,
 {
   background-color: var(--color-primary);
   border-radius: var(--remotion-themed-dropdown-scrollbar-border-radius);
-}
\ No newline at end of file
+}
+
+.remotion-themed[disabled] img,
```

```suggestion
button.remotion-themed:disabled img,
```
you can't nest img inside input or select or textarea, so it's just button:disabled.

There is no `disabled` attribute as a general concept in HTML. ASP.NET does it, but this is invalid markup.